packages/edit-site/src/components/global-styles/hooks.js

```diff
@@ -201,6 +201,11 @@ export function getSupportedGlobalStylesPanels( name ) {
 		supportKeys.push( 'blockGap' );
 	}
 
+	// check for shadow support
+	if ( blockType?.supports?.preview ) {
```

Comment:
Can we just assume this from the presence of an `example` construct?

Reply:
Thanks for the review. Component preview is now based on the presence of example 👍